.circleci/config.yml

```diff
@@ -328,7 +328,8 @@ executors:
     environment:
       <<: *common_environment
       CACHE_NONCE: "Debug"
-      OVERLAY_MIXINS: "debug ccache coverage-gcc"
+      OVERLAY_BUILD_MIXINS: "debug ccache coverage-gcc"
+      OVERLAY_TEST_MIXINS: "coverage-gcc"
```

Comment:
Is this needed, it doesn't seem to have a test verb equivalent? BTW, do we have any Python tests that we want coverage reports for?
https://github.com/colcon/colcon-mixin-repository/blob/1ddb69bedfd1f04c2f000e95452f7c24a4d6176b/coverage.mixin#L14

Reply:
What is needed? Because right now, it fails (look at the 3 most recent PRs) due to `ccache debug release` not existing for colcon `test`. The CI job below is the first to even get to a point of running tests

Reply:
https://github.com/ros-planning/navigation2/search?q=pytest&unscoped_q=pytest from that search, maybe? Unclear

Reply:
The debug CI is failing has coconut errors out when given a mixing that doesn't have support for the verb, shown in the mixin file linked above. That's why I suggested merely omitting the mixin argument from the colcon test invocation.

```
colcon: error: Mixin 'coverage-gcc' is not available for 'test'
```

Reply:
Ok

Reply:
I'll keep the variable but set as empty in case that changes and we need some test mixins. @ruffsl if you don't think that day will come, I can also remove it, let me know

Reply:
I'd suggest reverting the variable name to stay consistent with everything else, and omit the test mixin env. If we ever do need explicit test mixins in the future, I'll rework the CI accordingly.

Reply:
OK. I want to let this CI cycle run its course to see if that fixes this issue in both the builds (with empty string) and then I'll make those changes.